### ISSUE TITLE
Power management services are not enabled in Manjaro builds

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#165]
Both the `auto-cpufreq` and `tlp` services do not get enabled. This issue appears to be specific to Manjaro and not seen on Arch Linux builds.

**Comment by LukeShortCloud** [comment]
Here is the relevant log:

```
Failed to enable unit, unit /etc/systemd/system/auto-cpufreq.service is masked.
```

We already run `systemctl unmask auto-cpufreq` beforehand but it does not help.

**Comment by LukeShortCloud** [comment]
Based on the README of the auto-cpufreq project, it sounds like it's best to either use it or TLP. Not both. One benefit is that auto-cpufreq can turbo the CPU frequency whereas TLP cannot.

```
Using tools like [TLP](https://github.com/linrunner/TLP) can help in this situation with extending battery life (which is something I used to do for numerous years), but it also might come with its own set of problems, like losing turbo boost.

With that said, I needed a simple tool which would automatically make "cpufreq" related changes, save battery like TLP, but let Linux kernel do most of the heavy lifting. That's how auto-cpufreq was born.

Please note: auto-cpufreq aims to replace TLP in terms of functionality and after you install auto-cpufreq it's recommended to remove TLP. If both are used for same functionality, i.e: to set CPU frequencies it'll lead to unwanted results like overheating. Hence, only use [both tools in tandem](https://github.com/AdnanHodzic/auto-cpufreq/discussions/176) if you know what you're doing.
```

https://github.com/AdnanHodzic/auto-cpufreq

tl;dr? Only install and use `auto-cpufreq`.
